### PR TITLE
Integrate styling for sommerretreat back into master

### PR DESCRIPTION
**Pull request by greimela**
- [x] Figure out what to keep and what to dismiss

### CODE REVIEW

File: server/office.json [hunk 1]
```diff
@@ -89,5 +93,10 @@
       "groupId": "lord",
       "icon": "https://cdn.pixabay.com/photo/2015/11/03/09/03/meeting-1019995_960_720.jpg"
     }
-  ]
+  ],
+  "clientConfig": {
```
Comment (greimela) [inline server/office.json:97]: @klassm @endoval Could it be that this part of the office.json is read, but never used?

Comment (klassm) [reply server/office.json:97]: Yes I guess that changed back to env variables.

**Comment by greimela** [comment]
@endoval Could you sign-off the commits blamed by the DCO bot?

**Comment by greimela** [comment]
I'd say we keep everything from this branch, even the styling changes are very useful in general.

**Pull request merged**